node/src/chain_spec/turing.rs

```diff
@@ -56,7 +56,25 @@ pub fn turing_development_config() -> ChainSpec {
 			let initial_balance: u128 = ALLOC_TOKENS_TOTAL / accounts.len() as u128;
 			let endowed_accounts: Vec<(AccountId, Balance)> =
 				accounts.iter().cloned().map(|k| (k, initial_balance)).collect();
-
+			// Create endowed Tokens accounts starting with index 1.  Index 0 is Native and Native tokens get stored in Balances Pallet.
+			// 0 => Native,
+			// 1 => KSM,
+			// 2 => AUSD,
+			// 3 => KAR,
+			// 4 => LKSM,
+			// 5 => HKO,
+			// 6 => SKSM,
+			// 7 => PHA,
+			// 8 => UNIT,
+			let tokens_endowed_accounts: Vec<(AccountId, CurrencyId, Balance)> = accounts
+				.iter()
+				.cloned()
```

Comment:
I think this `cloned()` is unnecessary since you have to call `k.clone()` in the map anyway.